tracer/src/Datadog.Trace/ClrProfiler/InstrumentationFilter.cs

```diff
@@ -0,0 +1,15 @@
+// <copyright file="InstrumentationFilter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal enum InstrumentationFilter
```

Comment:
nit: Naming isn't highest priority, but when I was reviewing this I was thinking this enum could be named `InstrumentationScenario` or `InstrumentationCategory`, and the two enums could be `Tracing` and `AppSec`. Do you have any thoughts on those names? What you have right now is perfectly fine but I wanted to throw out some feedback.